src/util/entities/Template.ts

```diff
@@ -36,7 +36,7 @@ export class Template extends BaseClass {
 	source_guild_id: string;
 
 	@JoinColumn({ name: "source_guild_id" })
-	@ManyToOne(() => Guild)
+	@ManyToOne(() => Guild, { onDelete: "CASCADE" })
```

Comment:
this should be `onDelete: "NO ACTION"`.

Reply:
Does Discord.com not delete guild templates when the source guild is removed? Rory and I thought it did